### ISSUE TITLE
Improve editor controls tab navigation

### ISSUE THREAD

**Issue by bastianjoel** [OpenSlides/openslides-client#4150]
Currently when navigation with `<tab>` through a form with an `os-editor` one has to navigate over all format control buttons. 

**Wanted behavior**
When navigation with tab only the first format control should be focused. When hitting tab again the editors content should be focused. 
The other format buttons should still be reachable by using the keyboards arrow keys. 

The implementation should follow https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/

**Comment by reiterl** [comment]
I can include a tabindex="-1" for all elements in the os-editor. That should fix the tab navigation.

> The other format buttons should still be reachable by using the keyboards arrow keys.
How should that work? Should there be a special keydown  handler, which switchs the focus?

**Comment by bastianjoel** [comment]
> I can include a tabindex="-1" for all elements in the os-editor. That should fix the tab navigation.

Stick to the specification I provided.

**Comment by reiterl** [comment]
The problem with the arrow keys: Should that be solved with a kind of js event handler?

**Comment by reiterl** [comment]
Yes, it should work with a handler and the tabindex should be set to 0 for focused buttons.
